spec/function.spec.js

```diff
@@ -0,0 +1,127 @@
+const {
+  resetConstants,
+  stringToArray,
+  multiplyCheck,
+  findLetterScore,
+  scoreMultiplicatorizer,
+  tallyScore
+} = require('../src/scrabble')
+
+describe('stringToArray', () => {
+  beforeEach(() => {
+    resetConstants()
+  })
+
+  it("return ['H', 'I'] for 'hi'", () => {
+    expect(stringToArray('hi')).toEqual(['H', 'I'])
+  })
+
+  it("return ['G', 'O', 'O', 'D', ' ', 'D', 'A', 'Y'] for 'hi'", () => {
+    expect(stringToArray('good day')).toEqual([ 'G', 'O', 'O', 'D', ' ', 'D', 'A', 'Y']) // prettier-ignore
+  })
+
+  it('return empty string in array for 33', () => {
+    expect(stringToArray(33)).toEqual([''])
+  })
+
+  it('return empty string in array for null', () => {
+    expect(stringToArray(null)).toEqual([''])
+  })
+
+  it('return empty string in array for undefined', () => {
+    expect(stringToArray(undefined)).toEqual([''])
+  })
+
+  it('return empty string in array for {}', () => {
+    expect(stringToArray({})).toEqual([''])
+  })
+
+  it('return empty string in array for "("', () => {
+    expect(stringToArray('(')).toEqual(['('])
+  })
+})
+
+describe('multiplyCheck', () => {
+  beforeEach(() => {
+    resetConstants()
+  })
+
+  it("return false for 'c'", () => {
+    expect(multiplyCheck('c')).toBeFalse()
+  })
+
+  it("return true for ']'", () => {
+    expect(multiplyCheck(']')).toBeTrue()
+  })
+
+  it("return false for '('", () => {
+    expect(multiplyCheck('(')).toBeFalse()
+  })
+
+  it("return false for '()'", () => {
+    expect(multiplyCheck('()')).toBeFalse()
+  })
+
+  it("return false for '()'", () => {
+    expect(multiplyCheck('()')).toBeFalse()
+  })
+})
+
+xdescribe('findLetterScore', () => {
+  beforeEach(() => {
+    resetConstants()
+    scoreMultiplicatorizer() // TODO: wanted to test function with MULTIPLE but it always ends up = 1, can't figure where it goes :(
```

Comment:
Did you resolve this and just forgot the comment?

Reply:
No, hasn't been resolved. The first test is only returning `1` rather than `2` as it should be doing. Putting it down to not knowing exactly how Jasmine is working really.